src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -132,7 +132,9 @@ public override async Task<bool> Execute()
 
                 do
                 {
-                    if (!await this.PromptPackageIdentifierAndCheckDuplicates(manifests))
+                    if (this.WingetRepoOwner == DefaultWingetRepoOwner &&

```

Comment:
Why only for the main repo?

Reply:
If a user is attempting to generate a new package manifest for their own test repo, we shouldn't block them from creating manifests or testing out manifest creation. I believe we should only apply the duplicate check if they're using the main repo.

Reply:
If a user is attempting to generate a new package manifest for their own test repo, we shouldn't block them from creating manifests or testing out manifest creation. I believe we should only apply the duplicate check if they're using the main repo.

Reply:
You're probably right, though this won't always be for "test" repos, eventually users may have their own alternate "prod" repos I believe, but anyway this is fine for now I think